src/components/js/Home.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import "../css/home.css"
+
+function Home() {
+  return (
+    <div>
+      
+      <div className='bg-image'>
+        <img className="home-image" src='../images/home-img.jpg' /> 
+
+        <a className='text-on-image' href='#'> <h1> Explore ! </h1> </a>
+       </div>
```

Comment:
You should use `Link` from react-router to navigate to other pages of your app. `a` tag has to be used only for external links.